lockup-factory/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "lockup-factory"
-version = "0.2.0"
```

Comment:
Please bump the version after we will finally release the `lockup` contract

Reply:
Already bumped in here to 0.2.1 https://github.com/near/core-contracts/pull/158/commits/862c3a1b28f92184382801612ea0e703c8a06e14